### PR TITLE
Enterprise v open source table

### PR DESCRIPTION
**Pull request by davidrichards-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/getting-started/installation.rst [hunk 1]
```diff
@@ -16,8 +16,63 @@ You need to install:
...
+   * - High Throughput via Microservices
+     - Yes
+     - No
+   * - Resource Management
+     - Yes
+     - No
+   * - `Ledger Pruning <https://docs.daml.com/canton/usermanual/operational_processes.html#ledger-pruning>`_
+     - Yes
+     - No
+   * - Postgres Backend
```
Comment (bame-da) [inline docs/source/getting-started/installation.rst:54]: Backend vs driver. Maybe split out "local contract store in PostgreSQL" and "PostgreSQL driver"?

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -16,8 +16,63 @@ You need to install:
...
+   * - `Transaction Processing <https://docs.daml.com/canton/architecture/overview.html#node-scaling>`_
+     - Parallel (fast)
+     - Sequential (slow)
+   * - `High Availability <https://docs.daml.com/canton/usermanual/ha.html>`_
+     - Yes
+     - No
+   * - High Throughput via Microservices
+     - Yes
+     - No
+   * - Resource Management
```
Comment (bame-da) [inline docs/source/getting-started/installation.rst:48]: What's this? Is there documentation for this?

File: docs/source/getting-started/installation.rst [hunk 3]
```diff
@@ -16,8 +16,63 @@ You need to install:
...
+   * - Sub-Transaction Privacy
+     - Yes
+     - Yes
+   * - `Transaction Processing <https://docs.daml.com/canton/architecture/overview.html#node-scaling>`_
+     - Parallel (fast)
+     - Sequential (slow)
+   * - `High Availability <https://docs.daml.com/canton/usermanual/ha.html>`_
+     - Yes
+     - No
+   * - High Throughput via Microservices
```
Comment (bame-da) [inline docs/source/getting-started/installation.rst:45]: Is there documentation for the "microservices" deployment?

**Review by bame-da — APPROVED** [review]
Left a few comments that would be good to address. Feel free to merge once you think it's good enough.

File: docs/source/getting-started/installation.rst [hunk 4]
```diff
@@ -49,6 +104,9 @@ Installing Daml Enterprise
 If you have a license for Daml Enterprise, you
 can install it as follows:
 
+
+- Canton can be downloaded from the respective `repository <https://digitalasset.jfrog.io/artifactory/canton-enterprise/>`_
```
Comment (carrielaben-da) [inline docs/source/getting-started/installation.rst:108]:
```suggestion
- Canton can be downloaded from this `repository <https://digitalasset.jfrog.io/artifactory/canton-enterprise/>`_
```

**Review by carrielaben-da — COMMENTED** [review]
Just a couple of small changes.

File: docs/source/getting-started/installation.rst (hunk 3 above)
Comment (davidrichards-da) [reply docs/source/getting-started/installation.rst:45]: I'm changing this to 'Horizontal scalability' as requested on the slack conversation [here](https://daholdings.slack.com/archives/C973Z3TK2/p1648645300831909) and linking to [here](https://www.canton.io/docs/dev/user-manual/usermanual/ha.html#sequencer)

File: docs/source/getting-started/installation.rst (hunk 2 above)
Comment (davidrichards-da) [reply docs/source/getting-started/installation.rst:48]: I'm removing this row and if anyone from the Canton team wants to add it back in then they can do that in a separate PR.

File: docs/source/getting-started/installation.rst (hunk 1 above)
Comment (davidrichards-da) [reply docs/source/getting-started/installation.rst]: Done

**Pull request merged**